ReactWindows/ReactNative.Shared/ReactRootView.cs

```diff
@@ -201,6 +199,23 @@ protected override Size MeasureOverride(Size availableSize)
             return result;
         }
 
+        internal void StartTouchHandling()
+        {
+            if (TouchHandler == null)
```

Comment:
Is there some instance where `StartTouchHandling` can be called twice?

Reply:
Not really, or not without having a StopTouchHandling call happening in between (for the reload context scenario).
But I don't trust time and space anymore...

---
In reply to: [215817511](https://github.com/Microsoft/react-native-windows/pull/1958#discussion_r215817511) [](ancestors = 215817511)